mis_builder_contract/sql/mis_contract_line.sql

```diff
@@ -0,0 +1,30 @@
+CREATE OR REPLACE VIEW mis_contract_line AS (
```

Comment:
Instead of view, you may use the new syntax like https://github.com/odoo/odoo/blob/f9ba2ca30c77b6a24ea134410d30222a69d3ae6b/addons/account/report/account_invoice_report.py#L71-L73